deploy/docker/server.py

```
@@ -225,8 +269,10 @@ async def get_token(req: TokenRequest):
 async def config_dump(raw: RawCode):
     try:
         return JSONResponse(_safe_eval_config(raw.code.strip()))
+    except ValueError as e:
+        raise HTTPException(400, f"Configuration validation error: {str(e)}")
```

Ruff (0.13.1)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)